src/libraries/System.Security.AccessControl/pkg/System.Security.AccessControl.pkgproj

```diff
@@ -2,12 +2,8 @@
   <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.props))" />
   <ItemGroup>
     <ProjectReference Include="..\ref\System.Security.AccessControl.csproj">
-      <SupportedFramework>net461;netcoreapp2.0;$(AllXamarinFrameworks)</SupportedFramework>
+      <SupportedFramework>net461;netcoreapp2.0;uap10.0.16299;$(AllXamarinFrameworks)</SupportedFramework>
     </ProjectReference>
-    <File Include="$(PlaceHolderFile)">
-      <TargetPath>runtimes/win/lib/uap10.0.16299</TargetPath>
-    </File>
-    <InboxOnTargetFramework Include="uap10.0.16299" />
```

Comment:
why are removing the placeholder for uap?

Reply:
@ericstj might be able to better answer this one

Reply:
These packages were actually broken for UAP in last release (and maybe even previous) because the netstandard ref was versioned while UAP still got a placeholder (and thus would provide the older version inbox assembly).  The use of place holders is a relic of the old pre-conflict-resolution way of dealing with assemblies that were part of the framework.  We can remove that placeholder and simplify these cases while fixing the netstandard consumption case.  Worst case that happens here is that the implementation provided by the package is somehow worse than an old UAP configuration that was in the meta-package itself.  That's actually no worse than the type of breaks we already made when removing all the UAP package configurations from the repo.